### DISCUSSION TITLE
Boids experiment

### DISCUSSION THREAD

**Discussion by nowakpe** [nxrighthere/UnrealCLR#49] [Showcase]
https://www.linkedin.com/posts/petr-nowak-513198156_unrealengine-ue4-unrealclr-activity-6723503591308464128-kZyp

**Comment by nxrighthere** [comment]
I think with Niagara particles it's possible to scale such simulations even more. [Here's](https://www.youtube.com/watch?v=MoA5vfiL-Ws) a video with 600k meshes.

**Reply by nowakpe** [reply]
Sure, but it is probably everything computed on GPU. I was just trying to do some ECS stuff :)
